lib/modules/platform/bitbucket/__snapshots__/index.spec.ts.snap

```diff
@@ -140,6 +140,15 @@ exports[`modules/platform/bitbucket/index initRepo() works with username and pas
 }
 `;
 
+exports[`modules/platform/bitbucket/index massageMarkdown() removes create all rate-limited PRs at once text 1`] = `
+"## Rate-Limited
+These updates are currently rate-limited. Click on a checkbox below to force their creation now
+ - [ ] <!-- unlimit-branch=renovate/ansi-regex-6.x -->Update dependency ansi-regex to v6
```

Comment:
shouldn't it also remove these check boxes?

Reply:
i dont get you, what do you mean by "These" i posted a screenshot, this is the only problematic line i can see

Reply:
somehow  bitbucket doesn't produce them, i'm not sure why or how, 
they are not being replaced by anything
yet look at the screenshot, they look perfectly fine, i think we have extra massaging somewhere since 
the only message i can see is 
`These updates are currently rate-limited`
and i cant see the rest see screenshot

Reply:
i'm seriously confused now, i cant find where it is being removed xD

Reply:
i think i put a github issue input in the test instead of a bitbucket one ill recheck

Reply:
I don't see why anything needs removing unless the checkbox implies that action will take place immediately

Reply:
its a wrong test regardless.

@rarkins 
do you mean in total i should close this task completely? and it is ok for customers to see bitbucket with an extra comment that is meaningless?

Reply:
Why is it meaningless?

Reply:
As in.. checkboxes don't work?

Reply:
it says 🔐 **Create all rate-limited PRs at once** 🔐';
with an uncheckable [ ]

Reply:
So then all checkboxes should be removed if none of them are checkable?

Reply:
yes, that's my intention.

Reply:
yes exactly, 
i've changed the test to do that 
i had an input from a github issue on the unit test instead from a bitbucket issue

Reply:
see https://bitbucket.org/philipabed/test-renovate-body1/issues/1/dependency-dashboard for example